database/layer.go

```diff
@@ -25,13 +25,13 @@ import (
 )
 
 const (
-	FieldLayerIsValue           = "layer"
+	fieldLayerIsValue           = "layer"
 	FieldLayerID                = "id"
 	FieldLayerParent            = "parent"
 	FieldLayerSuccessors        = "successors"
 	FieldLayerOS                = "os"
-	FieldLayerInstalledPackages = "adds"
-	FieldLayerRemovedPackages   = "removes"
+	fieldLayerInstalledPackages = "adds"
+	fieldLayerRemovedPackages   = "removes"
 	FieldLayerEngineVersion     = "engineVersion"
```

Comment:
What's the reasoning behind selectively exposing these fields? Shouldn't it be all or nothing?